app/assets/scripts/views/analytics-admin-area.js

```diff
@@ -27,11 +27,16 @@ var AnalyticsAA = React.createClass({
     const done = ids.filter(v => v.inTheDatabase).length;
     const total = ids.length;
     const completion = ((done / total) * 100);
+    const bucket = 'https://s3.amazonaws.com/openroads-vn-dumps/by-province-name/';
```

Comment:
Information like bucket name here should be included in the config https://github.com/orma/openroads-vn-analytics/blob/develop/app/assets/scripts/config/production.js